### PR TITLE
docs: delegation instead of third party in declarative pn

### PR DESCRIPTION
**Pull request by vrolland**

### CODE REVIEW

File: packages/advanced-logic/specs/payment-network-any-declarative-0.2.0.md [hunk 1]
```diff
@@ -347,6 +355,88 @@ The 'addPaymentInstruction' event:
 
 ---
 
+#### addPayeeDelegate
```
Comment (benjlevesque) [inline packages/advanced-logic/specs/payment-network-any-declarative-0.2.0.md:358]: the 2 methods could be merged in 1 `addDelegate`, which would either add a payee or payer delegate depending on who did the action

**Review by benjlevesque — APPROVED** [review]
Not sure if this should be specified, but the delegates should probably be in the Topics of the Request

**Review by benjlevesque — APPROVED** [review]

File: packages/advanced-logic/specs/payment-network-any-declarative-0.2.0.md [hunk 2]
```diff
@@ -347,6 +355,50 @@ The 'addPaymentInstruction' event:
...
+- The signer is the `payee` and the extension property `payeeDelegate` is undefined
+- The signer is the `payer` and the extension property `payerDelegate` is undefined
+
+##### Warnings
+
+None.
+
+##### Results
+
+A extension state is updated with the following properties:
```
Comment (yomarion) [inline packages/advanced-logic/specs/payment-network-any-declarative-0.2.0.md:383]:
```suggestion
An extension state is updated with the following properties:
```

**Review by yomarion — COMMENTED** [review]
Just to be sure, the file will actually be renamed 0.3.0, correct ?

**Comment by vrolland** [comment]
> Not sure if this should be specified, but the delegates should probably be in the Topics of the Request

I think it should not be specified here, but you are right I should be in the code !

**Review by yomarion — APPROVED** [review]

**Pull request merged**